ces/commands.py

```diff
@@ -588,7 +590,11 @@ class SellCommand(PlaceOrderBaseCommand):
 
 Another example, selling all of our units of ETH at 1 BTC each:
 
-{0} BTC ETH amount max rate 1'''
+sell BTC ETH amount max rate 1
```

Comment:
Can you keep the `{0}` in these examples? The command's name will be replaced. This is so that if eventually a command was to be renamed, it could be done without having to do anything besides changing the name.

Reply:
Yep, will fix this covering all the examples. Thanks!